app/views/Mails/acceptrefuse/sendProposalRefused.scala.txt

```diff
@@ -2,39 +2,39 @@
 --------------------------------------------------------------------
 About your submission for @Messages("longYearlyName")
 --------------------------------------------------------------------
-FRENCH (ENGLISH VERSION FOLLOWS)
+ENGLISH (FRENCH VERSION FOLLOWS)
 
-Bonjour,
-
-Après plusieurs jours de sélection intense, nous avons le regret de vous informer que votre proposition de talk n'a pas été acceptée.
+Hi,
 
-Pour rappel : @proposal.title au format @Messages(proposal.talkType.id).
+After several days reviewing your submission for @Messages("longYearlyName"), we're sorry to tell you that this talk didn't make the cut:
+@proposal.title, @Messages(proposal.talkType.id).
 
-Nous vous remercions cependant pour le temps que vous avez consacré à préparer votre sujet et pour tout l'intérêt que vous avez porté à notre conférence ! En gage de reconnaissance, nous vous proposons de vous inscrire avec le code promotionnel cfp-invite. Vous bénéficierez d’un ticket réduit au prix early bird. RDV sur @ConferenceDescriptor.current().conferenceUrls.registration
+We would like to acknowledge your commitment and dedication and thank you a million times for the effort you put in. 
+If you have posted other proposals and do not have yet received any email for them, it is because they are still being reviewed. In the other case, and as we would still be happy to see you there, you will receive early February a discount code for you to use. No hard feelings! 
```

Comment:
If you have posted other proposals and **haven't received any email about those yet**

Reply:
Done